docs/backup_and_restore.rst

```diff
@@ -334,6 +338,74 @@ Migrating Using a V2+V3 or V3-Only Backup
 #. :doc:`Test the new instance <test_the_installation>` to verify that the
    web interfaces are available and the servers can be reached via SSH.
 
+.. _repair_admin_usbs:
+
+Repair Additional Admin Workstations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have additional *Admin Workstation* USBs, they will no longer have
+valid SSH credentials and will need to be repaired. In these steps, the "primary
+*Admin Workstation*" is the one which you used to complete the above migration 
+process.
+
+#. Prepare a fresh :doc:`Transfer Device <set_up_transfer_and_export_device>`.
+   In this case, use LUKS encryption and set a strong (7-word diceware) 
+   passphrase, which you may record in your primary *Admin Workstation*
+   KeePassXC password manager. 
+
+#. Copy the following files from your primary *Admin Workstation* onto the
+   Transfer Device:
+   
+   - ``~/Persistent/securedrop/install_files/ansible-base/tor_v3_keys.json``
+   - ``~/Persistent/securedrop/install_files/ansible-base/mon-ssh.auth_private``
+   - ``~/.ssh/id_rsa.pub``
+   - ``~/.ssh/id_rsa``
+
+#. Boot into each additional Admin Workstation, setting an administrative 
+   passphrase during startup. Once logged in, attach the Transfer Device
```

Comment:
I would say "set [an administration password](https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/) and unlock the persistent volume on the Tails welcome screen" (that's more consistent with the language we've used elsewhere in the docs).